packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -92,6 +100,11 @@ contract SuperchainConfig is Initializable, ISemver {
         guardian_ = Storage.getAddress(GUARDIAN_SLOT);
     }
 
+    /// @notice Getter for the upgrader address.
+    function upgrader() public view returns (address upgrader_) {
```

Comment:
Missing return var natspec?

Reply:
It doesn't give much value in this small function